src/ol/overlay.js

```diff
@@ -92,6 +95,26 @@ ol.Overlay = function(options) {
 
   /**
    * @private
+   * @type {boolean}
+   */
+  this.autoPan_ = goog.isDef(options.autoPan) ? options.autoPan : true;
+
+  /**
+   * @private
+   * @type {olx.animation.PanOptions}
+   */
+  this.autoPanAnimation_ = /** @type {olx.animation.PanOptions} */
+      (goog.isDef(options.autoPanAnimation) ? options.autoPanAnimation : {});
```

Comment:
And how about this:

```js
  this.autoPanAnimation = goog.isDef(options.autoPanAnimation) ? options.autoPanAnimation :
      /** @type {olx.animation.PanOptions} */ ({});
```

The cast is only needed when `options.autoPanAnimation` is not defined I guess.